lib/Fmt.mli

```diff
@@ -57,6 +54,26 @@ val break : int -> int -> t
 val force_break : t
 (** [force_break] forces a break with indentation 0. Equivalent to [break 1000 0].*)
 
+val space_break : t
+(** [space_break] is either a single space or a newline.
+    See {!Stdlib.Format.print_space_break}.
+    Equivalement to format syntax ["@ "].*)
+
+val cut_break : t
+(** [cut_break] is either a newline or a {!noop}.
+    See {!Stdlib.Format.print_cut}.
+    Equivalement to format syntax ["@,"].*)
+
+val flush_newline : t
```

Comment:
`flush_newline` doesn't occur in formatting code and is highly unwanted.

The only use, in `Translation_unit`, could as well be `force_newline`.

Reply:
Okay, I will change it !

Reply:
Let's merge!